.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+    - dependency-name: "joi"
```

Comment:
Joi is ignored since it might be breaking changes https://github.com/vlucas/frisby/pull/570